pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -315,6 +315,7 @@ def on_epoch_end(self, epoch, logs=None):
                                 f'\nEpoch {epoch:05d}: {self.monitor} reached',
                                 f'{current:0.5f} (best {self.best:0.5f}), saving model to',
                                 f'{filepath} as top {self.save_top_k}')
+                        self.best = current
```

Comment:
I think there's something wrong here. Perhaps this line should be removed?

Reply:
done